app/gui/src/dashboard/components/styled/FocusArea.tsx

```diff
@@ -46,12 +48,8 @@ function FocusArea(props: FocusAreaProps) {
   })
 
   const { focusWithinProps } = useFocusWithin({ onFocusWithinChange: onChangeFocusWithin })
-  const focusManager = useFocusManager()
-  const navigator2D = useNavigator2D()
```

Comment:
should `Navigator2D` be removed completely?

Reply:
and by extension `FocusArea`, `FocusRoot`, `FocusClassProvider`, `FocusDirectionProvider` (not sure if there's more)

Reply:
I think we still use it in assets table maybe?